airbyte-webapp/src/components/ui/PillSelect/PillButton.module.scss

```diff
@@ -38,11 +71,44 @@
   @include theme("red", colors.$red-50, colors.$red-600, colors.$red-100);
   @include theme("strongRed", colors.$red-300, colors.$white, colors.$red-400);
 
-  @include theme("disabled", colors.$grey-50, colors.$grey, colors.$grey-50);
+  @include disabled-theme("grey", colors.$grey-50, colors.$grey-600);
+  @include disabled-theme("blue", colors.$blue-50, colors.$blue-600);
+  @include disabled-theme("strongBlue", colors.$blue-500, colors.$blue-200);
+  @include disabled-theme("green", colors.$green-50, colors.$green-600);
+  @include disabled-theme("red", colors.$red-50, colors.$red-600);
+  @include disabled-theme("strongRed", colors.$red-300, colors.$white);
+
+  @include divider-theme("grey", colors.$grey-100, colors.$grey-200);
```

Comment:
Could the divider theme be combined with the theme? What are the color differences between the two?

Reply:
I decided to split those to make the styles look cleaner. in case if we pass all inside one theme, the theme itself will look like this `@mixin theme($name, $background, $text, $hover, $dividerBackground, $dividerHover, $disabledBackground, $disabledText)` and theme call will look like this `@include theme("grey", colors.$grey-50, colors.$grey-600, colors.$grey-100, colors.$grey-500, colors.$grey-600, colors.$grey-100, colors.$grey-200);` which is in my opinion not readable at all. @edmundito @dizel852 what do you think?

Reply:
I'm ok leaving it separated.